<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyDialog</class>
 <widget class="QDialog" name="MyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EventLoop Tester</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>385</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>150</height>
      </size>
     </property>
     <property name="title">
      <string>Param√®tres</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>276</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QCheckBox" name="chbox_kb_events">
         <property name="text">
          <string>Catch Keyboard Events</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chbox_pri_kb">
         <property name="text">
          <string>Priorty</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>361</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QCheckBox" name="chbox_timer_events">
         <property name="text">
          <string>Activate Timer Events</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spin_timer">
         <property name="maximum">
          <number>50000</number>
         </property>
         <property name="value">
          <number>150</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chbox_pri_timer">
         <property name="text">
          <string>Priority</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>30</y>
        <width>411</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Simple Msg</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_simple_msg">
         <property name="text">
          <string>testMsg</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_send">
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chbox_pri_msg">
         <property name="text">
          <string>Priority</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>130</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_close">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_close</sender>
   <signal>clicked()</signal>
   <receiver>MyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>742</x>
     <y>577</y>
    </hint>
    <hint type="destinationlabel">
     <x>695</x>
     <y>502</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
